test/integration-tests/test-utilities.ts

```diff
@@ -40,14 +40,14 @@ async function createTempDir(tempDir: string, folderName: string) {
  * Runs `sls deploy` for the given folder
  */
 function slsDeploy(tempDir: string): Promise<string> {
-  return exec(`cd ${tempDir} && $(npm bin)/serverless deploy`);
+  return exec(`cd ${tempDir} && serverless deploy`);
 }
 
 /**
  * Runs `sls remove` for the given folder
  */
 function slsRemove(tempDir: string): Promise<string> {
-  return exec(`cd ${tempDir} && $(npm bin)/serverless remove`);
```

Comment:
Why is that removed?

Reply:
$(npm bin) is not needed here and limits systems where this command can run

Reply:
`$(npm bin)` will use local packages without that command some of the global packages can be involved
Could you please revert it back?